proto/serverless/instrumentation/tags/v1/tags.proto

```diff
@@ -0,0 +1,306 @@
+syntax = "proto3";
+
+package serverless.instrumentation.tags.v1;
+
+option go_package = ".;protoc";
+
+message Tags {
+  // The required set of tags for The Serverless Ingest Platform. Payloads without all these tags will be rejected.
+  SlsSdkTags sls = 1;
+
+  // ============================================ //
+  // Defined TagSets start at field number 100  //
+  // ========================================== //
+
+  // ============================================ //
+  // Top level tags will have their underscores replaced //
+  // with periods when flattened. So for `aws_lambda` it will //
+  // be converted to `aws.lambda`//
+
+  // The root AWS Lambda Span tags.
+  optional AwsLambdaTags aws_lambda = 100;
+
+  // The AWS Lambda Initialization tags.
+  optional AwsLambdaInitializationTags aws_lambda_initialization = 101;
+
+  // The AWS Lambda Invocation tags.
+  optional AwsLambdaInvocationTags aws_lambda_invocation = 102;
+
+  // The AWS SDK Tags.
+  optional AwsSdkDynamodbTags aws_sdk_dynamodb = 103;
+  optional AwsSdkSqsTags aws_sdk_sqs = 104;
+  optional AwsSdkSnsTags aws_sdk_sns = 105;
+
+
+  // The HTTP Tags.
+  optional HttpTags http = 106;
+
+  // The HTTPS Tags.
+  optional HttpTags https = 107;
+
+  // The Express Tags.
+  optional ExpressTags express = 108;
```

Comment:
I think it'll be nicer to cover it under one property (e.g. `api_request`), that's defined with `HttpTags` and add there `framework_name` property

Reply:
Okay, we made a decision to have strictly defined. I'm fine to go this route but we need to commit as this will be the third time we have changed this.